<!doctype html>
<html lang="en">
  <head>
  	<title>NanoBaseLib</title>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
	<script src='https://kit.fontawesome.com/a076d05399.js' crossorigin='anonymous'></script>
    <link href="https://fonts.googleapis.com/css?family=Poppins:300,400,500,600,700,800,900" rel="stylesheet">
	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
	<link rel="stylesheet" href="css/style.css">
  </head>
  <body>
		
		<div class="wrapper d-flex align-items-stretch">
			<nav id="sidebar">
				<div class="img bg-wrap text-center py-3" style="background-image: url(images/bg.avif);">
	  			    <div class="user-logo">
	  				    <div class="img" style="background-image: url(images/logo_small.svg);"></div>
	  			     </div>
	  	        </div>
        <ul class="list-unstyled components mb-5">
                    <li>
                        <a href="index.html"><span class="fa fa-home mr-3"></span> Home </a>
                    </li>
                    <li>
                        <a href="dataset.html"><span class="fa fa-cloud-download mr-3 notif"></span> Dataset </a>
                    </li>
					<li>
                        <a href="pipeline.html"><span class="fa fa-sort-amount-desc mr-3"></span>  Pipeline </a>
                    </li>
					<li>
                        <a href="signal.html"><span class="fa fa-signal mr-3"></span> Signal Process </a>
                    </li>
                    <li>
                        <a href="basecall.html"> <span class="fa fa-line-chart mr-3"></span> Base Calling </a>
                    </li>
                    <li class="active">
                        <a href="ploya.html"><span class="fa fa-scissors mr-3"></span> PolyA Dection </a>
                    </li>
                    <li>
                        <a href="segment.html"><span class="fa fa-align-left mr-3"></span> Segmentation </a>
                    </li>
					<li>
                        <a href="mod.html"><span class="fa fa-hourglass-half mr-3"></span> Modification Dection </a>
                    </li>
                    <li>
                        <a href="software.html"><span class="fa fa-github mr-3"></span> Software </a>
                    </li>
                    <li>
                        <a href="about.html"><span class="fa fa-info-circle mr-3"></span> About </a>
                    </li>
                </ul>

    	</nav>

            <!-- Page Content  -->
            <div id="content" class="p-4 p-md-5 pt-5">
	         <h2 style="font-family:Georgia">PolyA tail dection benchmark</h2>	
		        <br/>			
                <figure>
				    <img src="images/polya.svg" width="60%" />
				    <figcaption>Illustration of Polyadenylation and its detection using Nanopore sequencing (Source of the top panel image: TT Weil, 2015.)</figcaption>
				</figure>		
                <br/>				
		        <p> <a href="https://en.wikipedia.org/wiki/Polyadenylation">Polyadenylation</a> is the addition of a poly(A) tail to an RNA transcript, typically a messenger RNA (mRNA). PolyA detection can serve as a quality control measure after base calling. Since the polyA tail is used in the sequencing library preparation, any sequence without a detectable polyA tail is flagged as a low-quality read and eliminated from downstream analysis. </p>
				<p>Download the benchmark datasets and resluts (<a href="https://462000561.lumidata.eu/nanobaselib/ont_ployA_standard.tar.gz.1">ont_ployA_standard.tar.gz</a>, <a href="https://462000561.lumidata.eu/nanobaselib/eGFP_polyA_DNA.tar.gz.1">eGFP_polyA_DNA.tar.gz</a>, and <a href="https://462000561.lumidata.eu/nanobaselib/eGFP_polyA_RNA.tar.gz.1">eGFP_polyA_RNA.tar.gz</a>).</p>
				<table class="noBorder" style="border: 0; border-collapse: collapse;">
	                <tr>
                        <th>  <i class="fa fa-circle-o"> </i>  </th>
			            <th>  <h3 style="font-family:Georgia"> &nbsp; PolyA detection rate benchmark </h3> </th>
			        </tr>
		        </table>
				We use seven datasets (shown in the x-axis) to benchmark the PolyA detection rate.
				<img src="images/polya_rate.svg" width="65%" />
				
				<br/>
				<br/>
				<table class="noBorder" style="border: 0; border-collapse: collapse;">
	                <tr>
                        <th>  <i class="fa fa-circle-o"> </i>  </th>
			            <th>  <h3 style="font-family:Georgia"> &nbsp; PolyA tail length estimation benchmark </h3> </th>
			        </tr>
		        </table>
				<p>The polyA tail length estimation distribution on dataset <a href="https://462000561.lumidata.eu/nanobaselib/ont_ployA_standard.tar.gz.1">ont_ployA_standard</a>.</p>
                <img src="images/ont_polyA_standard.svg" width="75%" />
				
				<table class="noBorder" style="border: 0; border-collapse: collapse;">
	                <tr>
                        <th>  <i class="fa fa-circle-o"> </i>  </th>
			            <th>  <h3 style="font-family:Georgia"> &nbsp; PolyA detection benchmark models introduction </h3> </th>
			        </tr>
		        </table>
				
				<h5 style="font-family:Georgia"> Nanopolish polya</h5> 
				<p>Nanopolish polya utilizes a Hidden Markov Model (HMM) to detect polyA tails and estimate their length. It divides each raw signal into five segments: start, leader, adapter, polyA tail, and transcript.</p>
				<center>
		            <img src="images/nanopolish_polya.png" width="50%" />
					<figcaption> An example of a squiggle segmentation generated by the hidden markov model. Distinct regions, from left to right: start (cyan), leader (yellow), <br/>  adapter (red), poly(A) tail (green),  and transcript (purple). Two samples flagged as “cliffs” can be observed in the poly(A) tail. (figure source: [2])</figcaption>
		        </center>	
				
				<h5 style="font-family:Georgia"> Tailfindr</h5> 
				<p> Tailfindr is an R tool that estimates poly(A) tail length from individual reads directly from ONT FAST5 raw data. tailfindr is able to estimate poly(A) tails from both RNA and DNA reads, including DNA reverse-complement reads containing poly(T) stretches.  [3]</p>
				<h5 style="font-family:Georgia"> Dorado</h5> 
				<p> Dorado is the latest basecaller developed by ONT, offering support for estimating polA tail length. </p>
				<br/>
				<table class="noBorder" style="border: 0; border-collapse: collapse;">
	                <tr>
                        <th>  <i class="fa fa-circle-o"> </i>  </th>
			            <th>  <h3 style="font-family:Georgia"> &nbsp; PolyA tail length ground truth label </h3> </th>
			        </tr>
		        </table>
				
				<p>For dataset <i>ont_ployA_standard</i>, the sample name is the label.</p>
				
				<p>While different datasets have different naming conventions. For dataset <i>eGFP_polyA_DNA</i> and <i>eGFP_polyA_RNA</i>, the sample names are organized by kit and replicate. And the label can be obtained using the barcode. The barcode in eGFP datasets is assigned by aligning the expected barcode sequences against the extracted read sequence preceding the eGFP alignment (see below Table).</p>
				<img src="images/tailfindr_barcode.png" width="50%" />
				<p>  </p>
				<img src="images/tailfindr_lable.png" width="50%" />
				<p>  </p>
				Download all the label files <a href="https://drive.usercontent.google.com/u/0/uc?id=1Pox89PxnJ5BlRf4_sZAAP0d5w6DAW-rA&export=download">tailfindr_label.tar.gz</a>.
				<br/>
				<br/>
				<h5 style="font-family:Georgia">Reference</h5>
				<p> [1] Weil T T. Post-transcriptional regulation of early embryogenesis[J]. F1000prime reports, 2015, 7. </p>
				<p> [2] Workman R E, Tang A D, Tang P S, et al. Nanopore native RNA sequencing of a human poly (A) transcriptome[J]. Nature methods, 2019, 16(12): 1297-1305. </p>
				<p> [3] Krause M, Niazi A M, Labun K, et al. tailfindr: alignment-free poly (A) length measurement for Oxford Nanopore RNA and DNA sequencing[J]. RNA, 2019, 25(10): 1229-1241. </p>
	            <center> <p>&copy; 2024. All rights reserved.</p> </center>
	        </div>  
	  
	</div>
</body>
</html>