<!doctype html>
<html lang="en">
  <head>
  	<title>NanoBaseLib</title>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
	<script src='https://kit.fontawesome.com/a076d05399.js' crossorigin='anonymous'></script>
    <link href="https://fonts.googleapis.com/css?family=Poppins:300,400,500,600,700,800,900" rel="stylesheet">
	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
	<link rel="stylesheet" href="css/style.css">
  </head>
  <body>
		
		<div class="wrapper d-flex align-items-stretch">
			<nav id="sidebar">
				<div class="img bg-wrap text-center py-3" style="background-image: url(images/bg.avif);">
	  			    <div class="user-logo">
	  				    <div class="img" style="background-image: url(images/logo_small.svg);"></div>
	  			     </div>
	  	        </div>
                <ul class="list-unstyled components mb-5">
                    <li>
                        <a href="index.html"><span class="fa fa-home mr-3"></span> Home </a>
                    </li>
                    <li>
                        <a href="dataset.html"><span class="fa fa-cloud-download mr-3 notif"></span> Dataset </a>
                    </li>
					<li>
                        <a href="pipeline.html"><span class="fa fa-sort-amount-desc mr-3"></span>  Pipeline </a>
                    </li>
					<li>
                        <a href="signal.html"><span class="fa fa-signal mr-3"></span> Signal Process </a>
                    </li>
                    <li>
                        <a href="basecall.html"> <span class="fa fa-line-chart mr-3"></span> Base Calling </a>
                    </li>
                    <li>
                        <a href="ploya.html"><span class="fa fa-scissors mr-3"></span> PolyA Dection </a>
                    </li>
                    <li class="active">
                        <a href="segment.html"><span class="fa fa-align-left mr-3"></span> Segmentation </a>
                    </li>
					<li>
                        <a href="mod.html"><span class="fa fa-hourglass-half mr-3"></span> Modification Dection </a>
                    </li>
                    <li>
                        <a href="software.html"><span class="fa fa-github mr-3"></span> Software </a>
                    </li>
                    <li>
                        <a href="about.html"><span class="fa fa-info-circle mr-3"></span> About </a>
                    </li>
                </ul>

    	    </nav>

            <!-- Page Content  -->

		
            <div id="content" class="p-4 p-md-5 pt-5">
	            <h2 style="font-family:Georgia">Segmentation & event alignment benchmark</h2>
	            <br/>
				<center>
		            <img src="images/Segmention.png" width="95%" />
					<figcaption>Illustration of Nanopore signal segmentation and event alignment.</figcaption>
		        </center>	
		        <br/>
				<p>After base calling, we can get a base-called sequence that will align with the reference sequence. The segmentation and event alignment task inputs the raw signal and reference sequence. It would segment the raw signal into small fragments and align it with the reference sequence. So one small fragment and its corresponding nucleotide would be called an "event".</p>
                <p>For each fragment of the raw signal, we can get its mean and standard deviation, which are used to evaluate the task.</p>
				<p>The benchmark results are available from the <a href="https://nanobaselib.github.io/dataset.html"> Dataset Download </a>.</p>
				
				<table class="noBorder" style="border: 0; border-collapse: collapse;">
	                <tr>
                        <th>  <i class="fa fa-circle-o"> </i>  </th>
			            <th>  <h3 style="font-family:Georgia"> &nbsp; Segmentation and event alignment benchmark models introduction </h3> </th>
			        </tr>
		        </table>
				
				<h5 style="font-family:Georgia"> Tombo re-squiggle</h5> 
				<p>The re-squiggle algorithm is the basis for the Tombo framework. The re-squiggle algorithm takes as input a read file (in FAST5 format) containing raw signal and associated base calls. The base calls are mapped to a genome or transcriptome reference and then the raw signal is assigned to the reference sequence based on an expected current level model. [1]</p>
				<center>
		            <img src="images/adaptive_half_z_scores.png" width="45%" />
					<figcaption>Full read adaptive banded shifted half-normal scores. (figure source: [1])</figcaption>
		        </center>	
				
				<h5 style="font-family:Georgia"> Nanopolish eventalign</h5> 
				<p> The hidden Markov model Nanopolish designed for the consensus problem had 5-mers of a proposed consensus sequence as the backbone of the HMM, with additional states and transitions to handle the skipping/splitting artefacts. Nanopolish used this HMM to calculate a consensus sequence from a set of reads. If we make a reference genome the backbone of the HMM, we can use it to align events to the reference. [2]</p>
				<h5 style="font-family:Georgia"> SegPore eventalign</h5> 
				<p> SegPore extends the model kmer table by estimating the distribution parameters for both the normal and modified states of each kmer.</p>
				<center>
		            <img src="images/segpore.png" width="40%" />
					<figcaption>Illustration of SegPore eventalign. (figure source: [3])</figcaption>
		        </center>
				
				<h5 style="font-family:Georgia">Reference</h5>
		        <p>[1] https://nanoporetech.github.io/tombo/resquiggle.html </p>
				<p>[2] https://simpsonlab.github.io/2015/04/08/eventalign </p>
				<p>[3] https://www.biorxiv.org/content/10.1101/2024.01.11.575207 </p>
		
		
		
	            <center> <p>&copy; 2024. All rights reserved.</p> </center>
	        </div>
	</div>
</body>
</html>